docker/launch-netbox.sh

```diff
@@ -22,27 +47,37 @@ load_configuration() {
   # this curl call will get a reply once unit is fully launched
   curl --silent --output /dev/null --request GET --unix-socket $UNIT_SOCKET http://localhost/
 
-  echo "⚙️ Applying configuration from $UNIT_CONFIG"
+  if [[ -n "$UNIT_CONFIG" ]] && [[ -s "$UNIT_CONFIG" ]]; then
+    echo "⚠️ The UNIT_CONFIG environment variable is deprecated. All *.pem and *.json files in /etc/unit will be loaded automatically when UNIT_CONFIG is undefined."
+    echo "⚙️ Applying configuration from UNIT_CONFIG environment variable: $UNIT_CONFIG"
```

Comment:
```suggestion
    echo "⚙️ Applying configuration from UNIT_CONFIG environment variable: '$UNIT_CONFIG'"
```

I dislike that the variable would become deprecated. I suggest to complement the variable with a new `$UNIT_PEM_FILE` variable that loads just that one certificate bundle – if defined.